### ISSUE TITLE
Misconfigured dependency?

### ISSUE THREAD

**Issue by danherrera** [TapTrack/TCMPTappy-Android#2]
Hello, hope you are having a great day/afternoon/evening! :-)

I added the following dependencies to an Android project without any issues:

```groovy
    implementation 'com.taptrack:tappyusb:2.4.0'
    implementation 'com.taptrack:tappy:2.4.0'
```

However, as soon as I introduce the following:
```groovy
    implementation 'com.taptrack:commandfamily-basicnfc:2.4.0'
```

It results in the following error:
```sh
Could not find TCMPTappy-Android:tcmp:unspecified.
Required by:
    project :app > com.taptrack:commandfamily-basicnfc:2.4.0
```

I tried adding all of the dependencies listed in the README in case it required one or a combination of them, but it resulted in the same error.

**Comment by dshalaby** [comment]
Hi there, thanks for you choosing the Tappy NFC readers for your project :)

I have replicated this issue - and am the first to admit that we've had some growing pains with our migration to Maven Central that we'll be looking into.   Thank you raising this ticket. 

For now, if it suits your project, I recommend using the versions of the command families that are still quite recent and published [here](https://mvnrepository.com/artifact/com.taptrack.tcmptappy2/commandfamily-basicnfc/2.3.0).  To do this replace:

` implementation 'com.taptrack:commandfamily-basicnfc:2.4.0'`

with

`implementation 'com.taptrack.tcmptappy2:commandfamily-basicnfc:2.3.0'`

This resolves the build error on my end. 

I invite you to email us at contact@taptrack.com for free support for your project - we're always interested in lending a hand to make your NFC project as painless as possible.  Thanks again for choosing Tappy readers!

**Comment by dshalaby** [comment]
This issue has now been resolved in version 2.4.1 which is now live. Thanks you for your patience and for bringing this issue to our attention.
